cfme/tests/control/test_basic.py

```diff
@@ -44,17 +45,181 @@
     # Needs working input/select mutability
 ]
 
-CONTAINER_POLICIES = [
+POLICIES = [
+    explorer.HostCompliancePolicy,
+    explorer.VMCompliancePolicy,
     explorer.ReplicatorCompliancePolicy,
     explorer.PodCompliancePolicy,
     explorer.ContainerNodeCompliancePolicy,
     explorer.ContainerImageCompliancePolicy,
+    explorer.HostControlPolicy,
+    explorer.VMControlPolicy,
     explorer.ReplicatorControlPolicy,
     explorer.PodControlPolicy,
     explorer.ContainerNodeControlPolicy,
     explorer.ContainerImageControlPolicy
 ]
 
+CONTROL_POLICIES = [
+    explorer.HostControlPolicy,
+    explorer.VMControlPolicy,
+    explorer.ReplicatorControlPolicy,
+    explorer.PodControlPolicy,
+    explorer.ContainerNodeControlPolicy,
+    explorer.ContainerImageControlPolicy
+]
```

Comment:
Okay, so if these are a subset of the whole list, do you think it would make sense to make the longer list a concatenation of the CONTROL_POLICIES and the rest of them?